.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "granulate-utils"]
 	path = granulate-utils
-	url = git@github.com:Granulate/granulate-utils.git
+	url = git@github.com:teoluwaf/granulate-utils.git
+	branch = py-win
+[submodule "py-spy"]
```

Comment:
Remove? (we don't need it)

Reply:
Remove the submodule - instead, clone the repository in a build script and pin it to the same hash as we use in the Linux build script. (actually, better, to ensure they both share the hash - add a file `scripts/pyspy_tag` and `scripts/pyspy_revision` which represent the git tag & revision used to build pyspy; to avoid cloning it)

Reply:
See note on dependencies recommendations

Reply:
Closing the discussion here, we continue in comments